### PR TITLE
Fix some translation strings

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

File: src/CommonGLPI.php [hunk 1]
```diff
@@ -1080,7 +1080,7 @@ class='btn btn-sm btn-icon btn-ghost-secondary " . ($prev >= 0 ? '' : 'bs-invisi
                 ]);
                 if ($this->isField('is_deleted') && $this->fields['is_deleted']) {
                     $title = $this->isField('date_mod')
-                                ? sprintf(__s('Item has been deleted on %1$s'), Html::convDateTime($this->fields['date_mod']))
+                                ? sprintf(__s('Item has been deleted on %s'), Html::convDateTime($this->fields['date_mod']))
```
Comment (trasher) [inline src/CommonGLPI.php:1083]: Why this change? Original should work, changing requires translation validation.

Comment (trasher) [reply src/CommonGLPI.php:1083]: Ok, it's a duplicate :)

**Review by trasher — APPROVED** [review]

File: src/CommonGLPI.php (hunk 1 above)
Comment (cedric-anne) [reply src/CommonGLPI.php:1083]: To use the same as here: https://github.com/glpi-project/glpi/blob/8fd103f268542027b178e91ba7bf98c74e73f811/templates/components/form/header_content.html.twig#L111
